.travis.yml

```diff
@@ -37,6 +37,11 @@ matrix:
     include:
         - php: 5.6
           env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" COVERAGE=true TEST_COMMAND="composer test-coverage"
+        - php: 7.3
```

Comment:
Hmm, just seen that this repo now relies on github actions. Anyone who can help me figure out how this action would look like?

Reply:
inkeeping with the strange usage of the `shivammathur/setup-php` action rather then docker images 🤷‍♂ , something like this:

```patch
--- a/.github/workflows/ci.yml
+++ b/.github/workflows/ci.yml
@@ -82,6 +82,34 @@ jobs:
       - name: Run tests
         run: composer test
 
+  static-analysis:
+    name: Build
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php: [ '7.3']
+
+    steps:
+      - name: Set up PHP
+        uses: shivammathur/setup-php@1.3.6
+        with:
+          php-version: ${{ matrix.php }}
+          extension-csv: bcmath, gmp, intl, dom, mbstring
+
+      - name: Checkout code
+        uses: actions/checkout@v1
+
+      - name: Download dependencies
+        run: composer update --prefer-dist --no-interaction
+
+      - name: Download Psalm
+        run: |
+          wget https://github.com/vimeo/psalm/releases/download/3.7.1/psalm.phar
+          chmod +x psalm.phar
+
+      - name: Run Static Analysis
+        run: psalm.phar
+
   docs:
     name: Docs
     runs-on: ubuntu-latest
```

Reply:
Thanks @bendavies  - exactly what I needed!